frontend/common/src/components/Alert/Alert.tsx

```diff
@@ -1,52 +1,131 @@
 import * as React from "react";
-import BasicAlert from "./BasicAlert";
-import LargeAlert from "./LargeAlert";
+import { XCircleIcon } from "@heroicons/react/24/outline";
+import { useIntl } from "react-intl";
+
+import type { HeadingLevel } from "../Heading";
+
+import {
+  styleMap,
+  iconStyleMap,
+  iconMap,
+  dismissStyleMap,
+  getAlertLevelTitle,
+} from "./utils";
 
 import "./alert.css";
 
-export type Color = "success" | "warning" | "info";
-export type AlertMode = "basic" | "large";
+export type AlertType = "success" | "warning" | "info" | "error";
+type AlertHeadingLevel = HeadingLevel | "p";
 
-export interface AlertProps {
-  title: React.ReactNode | string;
-  children: React.ReactNode;
-  icon: React.FC<React.SVGProps<SVGSVGElement>>;
-  type: Color;
-  mode?: AlertMode;
+export interface AlertProps
+  extends Omit<React.HTMLProps<HTMLDivElement>, "title"> {
```

Comment:
Could you please add some comments on these props?  Most are pretty clear but I'm not sure what "live" does and not 100% clear on heading level either.

Reply:
Great idea! Documented with [document alert props interface](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4668/commits/203c1b905cf1f0ee1b0b738290c38e9680ee489c)

Reply:
:man_cook: :pinched_fingers: